tools/torchci/td/get_merge_base_info.py

```diff
@@ -17,36 +17,24 @@
 
 FAILED_TEST_SHAS_QUERY = """
 SELECT
-    DISTINCT j.head_sha,
+    DISTINCT j.head_sha as head_sha
 FROM
-    commons.failed_tests_run t
-    join workflow_job j on t.job_id = j.id
-    left outer join commons.merge_bases mb on j.head_sha = mb.sha
+    default.failed_test_runs t
+    join default.workflow_job j on t.job_id = j.id
+    left anti join default.merge_bases mb on j.head_sha = mb.sha
```

Comment:
Any insights why `anti join` is a better choice here?

Reply:
anti join is an outer join that filters out results where the right side has no match.  I don't know if there are any performance benefits, but it does seem cleaner in terms of query syntax

Reply:
That sounds like the same as a "RIGHT JOIN" in normal SQL terminology?

Reply:
Sorry I mean that it only includes results where the right side has no match